### PR TITLE
Add IAM role support for kubernetes pods

### PR DESCRIPTION
**Pull request by kerin**
## What

- Adds [kube2iam](https://github.com/jtblin/kube2iam) daemonset Helm chart

## How to review

1. Install kube2iam Helm chart in environment
2. Create test IAM role in AWS with permissions to list an existing S3 bucket
3. Start [test kubernetes pod](https://github.com/jtblin/kube2iam#kubernetes-annotation) with `iam.amazonaws.com/role: role-name` annotation, with edited s3 command to list bucket
3. Check pod logs for S3 bucket listing

Provide [http://example.com](links) to relevant tickets, articles or other
resources.

Note the Trello Github integration links tickets to PRs/branches/commits (in
both directions).

### CODE REVIEW

File: charts/kube2iam/templates/_helpers.tpl [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{{/* vim: set filetype=mustache: */}}
```
Comment (andyhd) [inline charts/kube2iam/templates/_helpers.tpl]: Is this necessary?

Comment (kerin) [inline charts/kube2iam/templates/_helpers.tpl]: ¯\_(ツ)_/¯ that's a standard helper file generated by Helm. I'd say stick with it, given that all Helm helper templates have that.

**Comment by andyhd** [comment]
Is this ok to merge?

**Comment by kerin** [comment]
yes

**Pull request merged**